app/models/user.rb

```diff
@@ -397,6 +397,10 @@ def tag_moderator?
     roles.where(name: "tag_moderator").any?
   end
 
+  def self.moderator_availability_delay
```

Comment:
This seems to matter only in the context of the service, why not put it as a module constant there?

Reply:
I have moved this one to a module.